hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java

```diff
@@ -180,6 +182,18 @@ public String convertToLetter(int size) {
         return parameterCreator.getParameters();
     }
 
+    private void addJetParameters(PhoneHomeParameterCreator parameterCreator, BuildInfo buildInfo) {
+        if (buildInfo.getJetVersion() == null) {
+            parameterCreator.addParam("jetv", "");
+            parameterCreator.addParam("jetb", "");
+            parameterCreator.addParam("jetr", "");
```

Comment:
Could you update [PhoneHomeTest](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/test/java/com/hazelcast/util/PhoneHomeTest.java#L44) according to these changes? Default values are enough to test.